src/models/game.py

```diff
@@ -20,15 +45,56 @@ class Game:
     current_player_id: t.Optional[int]
     step_count: int
     winner_id: t.Optional[int]
-    field: t.Union[str, list]
+    field: t.Union[str, t.List[chr]]
     current_state: str
+    owner_mark: chr = 'X'
+    opponent_mark: chr = '0'
+
+    def _game_loop(self, field: t.List[str]):
```

Comment:
game loop?

Looks like it can be splitted into 2 separated functions,
1. Check if current mark is wins that returns bool
2. Update game according to result of 1-st function